docs/server-startup.md

```diff
@@ -96,6 +97,38 @@ The following is an example of how you might set both the port and the config:
     sudo ./startup.sh
     ```
 
+### CentOS Specific Steps
+
+For Python on CentOS you may need to rebuild it with enabling all the features.
+Also you'll need `pip` and `setuptools` for TabPy startup script to work.
+To install and enable all the required prerequisites follow the steps:
+
+```sh
+sudo yum update
+sudo yum groupinstall -y "Development tools"
+sudo yum install -y zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel expat-devel
+sudo yum install -y wget
+wget http://python.org/ftp/python/3.6.5/Python-3.6.5.tar.xz
+tar xf Python-3.6.5.tar.xz
+cd Python-3.6.5

```

Comment:
Just out of curiosity is there any reason to suggest installing Python from source here? Seems simpler to let the package manager take care of that?

Reply:
Valid point. These steps are specifically for Python 3.6.5. I believe when installing with `yum install python36u` the latest (3.6.8) is installed. I'll update the documentation.